dadjokes/src/Jokes/Jokes.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react';
+import axios from 'axios'
+
+
+class Jokes extends Component {
+    state = {
+        jokes: []
+    }
+
+    signout = () => {
+        localStorage.removeItem('jwt')
+        this.setState({ jokes: [] })
+    }
+    componentDidMount = () => {
+        const token = localStorage.getItem('jwt')
+        const endpoint = 'http://localhost:3300/api/jokes'
+        const options = {
+            headers: {
+                Authorization: token
+            }
+        }
+        this.forceUpdate();
```

Comment:
Are you calling `forceUpdate()` here to ensure you're component has the freshest options object?

Reply:
Actually this is left from something I was trying to do with the sign ina nd sign out. But you are right I was trying to see if this would re-render the application. I had the sign In and sign out nav link the same. But it would not re-render the entire app so you wouldn’t see sign out unless you clicked refresh. So I got tired of fighting and just created a new link.